ui_framework/src/components/info_button/info_button.js

```diff
@@ -6,14 +6,16 @@ import classNames from 'classnames';
 const KuiInfoButton = props => {
   const iconClasses = classNames('kuiInfoButton', props.className);
 
+  const ariaLabel = props['aria-label'] || 'Info';
```

Comment:
Instead of doing this we can create a `defaultProps` definition after the `propTypes` on line 17:

```javascript
KuiInfoButton.defaultProps = {
  'aria-label': 'Info',
};
```